pennylane_lightning/lightning_qubit/_measurements.py

```diff
@@ -411,7 +411,7 @@ def _process_single_shot(samples):
                         ).astype(int, copy=False)
                     else:
                         samples = self._measurement_lightning.generate_samples(
-                            len(wires), s
+                            list(wires), s
```

Comment:
Does this allow users to use customize wires, say `wires = ["alice", 3.14, -1, 0, "bob", "l", "m", "n"]`?

Reply:
Yes, I believe this is taken care of by `qml.map_wires`.